src/EnergyPlus/SurfaceGroundHeatExchanger.hh

```diff
@@ -259,6 +259,10 @@ namespace SurfaceGroundHeatExchanger {
         //==============================================================================
 
         void ReportSurfaceGroundHeatExchngr(EnergyPlusData &state); // Index for the surface under consideration
+
+        //==============================================================================
```

Comment:
For the record, you don't have to continue the formatting patterns like this, modern IDEs can present us the code in sufficiently nice ways. :)